/*
 * Copyright (C) Mellanox Technologies Ltd. 2001-2013.  ALL RIGHTS RESERVED.
 *
 * This software product is a proprietary product of Mellanox Technologies Ltd.
 * (the "Company") and all right, title, and interest in and to the software product,
 * including all associated intellectual property rights, are and shall
 * remain exclusively with the Company.
 *
 * This software is made available under either the GPL v2 license or a commercial license.
 * If you wish to obtain a commercial license, please contact Mellanox at support@mellanox.com.
 */


Bug Fix: Add a missing #endif.

--- lwip.orig/src/include/netif/etharp.h	2011-09-14 11:17:58.137138000 +0300
+++ lwip/src/include/netif/etharp.h	2011-09-14 11:19:10.017394000 +0300
@@ -212,10 +212,11 @@
 
 extern const struct eth_addr ethbroadcast, ethzero;
 
-#endif /* LWIP_ARP || LWIP_ETHERNET */
-
 #ifdef __cplusplus
 }
 #endif
 
+#endif /* LWIP_ARP || LWIP_ETHERNET */
+
+
 #endif /* __NETIF_ARP_H__ */
